axelor-production/src/main/java/com/axelor/apps/production/service/manuforder/ManufOrderServiceImpl.java

```diff
@@ -1280,4 +1285,39 @@ protected ManufOrder getParentMO(
     }
     return parentMO;
   }
+
+  @Override
+  public BigDecimal computeProducibleQty(ManufOrder manufOrder) throws AxelorException {
+    Company company = manufOrder.getCompany();
+    BillOfMaterial billOfMaterial = manufOrder.getBillOfMaterial();
+
+    if (company == null
+        || billOfMaterial == null
+        || billOfMaterial.getQty().compareTo(BigDecimal.ZERO) <= 0
+        || CollectionUtils.isEmpty(billOfMaterial.getBillOfMaterialSet())) {
+      return BigDecimal.ZERO;
+    }
+
+    BigDecimal producibleQty = null;
+    BigDecimal bomQty = billOfMaterial.getQty();
+
+    for (BillOfMaterial billOfMaterialLine : billOfMaterial.getBillOfMaterialSet()) {
+      Product product = billOfMaterialLine.getProduct();
+      BigDecimal availableQty =
+          productStockLocationServiceImpl.getAvailableQty(product, company, null);
+      BigDecimal qtyNeeded = billOfMaterialLine.getQty();
+      if (availableQty.compareTo(BigDecimal.ZERO) > 0 && qtyNeeded.compareTo(BigDecimal.ZERO) > 0) {
+        BigDecimal qtyToUse = availableQty.divideToIntegralValue(qtyNeeded);
+        producibleQty = producibleQty == null ? qtyToUse : producibleQty.min(qtyToUse);
+      }
+    }
+
+    producibleQty =
+        producibleQty == null
+            ? BigDecimal.ZERO
+            : producibleQty
+                .multiply(bomQty)
+                .setScale(appBaseService.getNbDecimalDigitForQty(), RoundingMode.UP);
```

Comment:
Please change to `RoundingMode.HALF_UP` since `RoundingMode.UP` will, for example, round `4.3` to `5`